src/main/scala/edu/berkeley/ce/rockslicing/LoadBalancer.scala

```diff
@@ -311,4 +315,127 @@ object LoadBalancer {
     assert(centroid1.length == 3 && centroid2.length == 3)
     (centroid1(0) < centroid2(0)) && (centroid1(1) < centroid2(1)) && (centroid1(2) < centroid2(2))
   }
+
+  private def excludeProcessorFace(faces: Seq[Face], absNormVec: ((Double, Double, Double), Double)): Seq[Face] = {
```

Comment:
Okay. It's private and a one-liner, so I added a brief comment rather than doing full Java doc.